tensorflow_lite_support/cc/task/processor/proto/detection_options.proto

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+syntax = "proto2";
+
+package tflite.task.processor;
+
+option java_multiple_files = true;
+option java_package = "org.tensorflow.lite.task.processor.proto";
+
+// Options for object detection processor.
+// Next Id: 6
+message DetectionOptions {
```

Comment:
Better to create a new one, in case we'll want to add new options for object detection in the future. And since we are working on merging MP solutions with Task Library, the two options protos will very likely become diverged.